### PR TITLE
Styling changes for open tickets

### PR DESCRIPTION
**Pull request by rememberlenny**
# What

Styling changes for :

#641 - Ryans must-haves before shipping
#597 - Preamble/heading
#624 - Footnotes at bottom of the page
#623 - Footnotes in tables
#564 - Tables
#600 - List items

--

screen shot 2017-11-15 at 11 05 57 pm

screen shot 2017-11-15 at 11 05 45 pm

screen shot 2017-11-15 at 11 03 53 pm

screen shot 2017-11-15 at 11 03 50 pm

screen shot 2017-11-15 at 11 03 43 pm

screen shot 2017-11-15 at 11 03 37 pm

screen shot 2017-11-15 at 11 03 31 pm

screen shot 2017-11-15 at 5 50 29 pm

### CODE REVIEW

**Comment by cmc333333** [comment]
Rather than trying to solve all six tickets (three of which have already been solved?) at once, let's try splitting this PR up. I'll add comments throughout, but let's start jigsawing this into rebased PRs for the (new) pieces.

File: ui/components/node-renderers/list-item.js [hunk 1]
```diff
@@ -4,8 +4,15 @@ import React from 'react';
 import renderNode from '../../util/render-node';
 
 export default function ListItem({ docNode }) {
+  let markerType;
+  if (docNode.marker == "•") {
+    markerType = "list-style-bullet";
+  } else if (docNode.marker.match(/\d+/g) != null) {
+    markerType = "list-style-number";
+  }
```
Comment (cmc333333) [inline ui/components/node-renderers/list-item.js]: I think doing this sort of logic in the frontend is going to be pretty error prone. The markers for lists will vary depending on our document; sometimes they'll have a parenthesis, sometimes they'll be letters, other times they'll be triangles. If it's really critical that we distinguish between numbered lists and bulleted lists, we'll want to change that at the structural level (i.e. create different node types).

File: ui/components/node-renderers/table/tfoot.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import renderContents from '../../../util/render-contents';
+import renderContent from '../../../util/render-contents';
```
Comment (cmc333333) [inline ui/components/node-renderers/table/tfoot.js]: We probably want to be consistent with this function name as `renderContents`.

**Comment by rememberlenny** [comment]
Resubmitting this as separate tickets

**Pull request closed without merging**